app/views/layouts/application.html.slim

```diff
@@ -9,7 +9,8 @@ html
 
     = favicon_link_tag
 
-    meta name="apple-itunes-app" content="app-id=#{Rails.application.secrets.mobile_app_id["ios"]}"
+    /Removed due to the implementation of a custom smart banner
+    /meta name="apple-itunes-app" content="app-id=#{Rails.application.secrets.mobile_app_id["ios"]}"
```

Comment:
I think it is, because it looks like that this meta tag only works on Safari, but I could leave this and not show the custom smart bannert on iOs/safari, what do you think?